### DISCUSSION TITLE
Link styles

### DISCUSSION THREAD

**Discussion by shuizhongyue120** [swup/swup#821] [Help]
pageA goto pageB; but，with diffrent link style; when back to PageA,the links style is losted

**Comment by daun** [comment]
Welcome. Can you share a bit more context? To get people to help you, they'll need more information about what you're trying to achieve and what's going wrong.

**Comment by hirasso** [comment]
I'd like to close this. Posts like these will burn out the people trying to help.

**Reply by daun** [reply]
True. Go ahead 🌝
